tools/c7n_azure/tests_azure/tests_resources/test_app_gateway.py

```diff
@@ -0,0 +1,30 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from ..azure_common import BaseTest
+
+
+class ApplicationGatewayTest(BaseTest):
+    def setUp(self):
+        super(ApplicationGatewayTest, self).setUp()
+
+    def test_app_gateway_validate(self):
+        with self.sign_out_patch():
```

Comment:
nit: iirc, we don't really need this `with`.. Might be wrong, was a long time ago.